### PR TITLE
[ONNX] norm builders should produce v1 ops

### PR DESCRIPTION
**Pull request by mbencer**
Current builders (which produce `v0` ops) were removed because they were used only by ONNX importer and FusedOps (`GRN` and `NormalizeL2`) which belong to `opset1`

### CODE REVIEW

**Comment by mbencer** [comment]
TESTNOW_DLDT_E2E

File: src/ngraph/builder/norm.cpp [hunk 1]
```diff
@@ -36,125 +37,147 @@ namespace ngraph
...
+                shared_ptr<Node> lp_norm(const Output<Node>& value,
+                                         size_t p_norm,
+                                         const AxisSet& reduction_axes,
+                                         float bias)
+                {
+                    // In general "entrywise" lp-norm for matrix `A` is defined as following double
+                    // sum:
+                    // ||A||_p = ||vec(A)||_p = [sum_{i=1}^m sum_{j=1}^n abs(a_{i,j})^p]^{1/p}
+                    shared_ptr<Node> abs_values{make_shared<ngraph::opset1::Abs>(value)};
+                    shared_ptr<Node> p_node = ngraph::opset1::Constant::create(
```
Comment (arogowie-intel) [inline src/ngraph/builder/norm.cpp]: We can create here just a scalar with `p_norm` value and later relay on NumPy autobroadcasting mechanism in `Power` op.

File: src/ngraph/builder/norm.cpp [hunk 2]
```diff
@@ -36,125 +37,147 @@ namespace ngraph
...
+
+                    // Get inner part of equation: abs_values^p_node, then sum over reduction_axes.
+                    shared_ptr<Node> values{make_shared<ngraph::opset1::Power>(abs_values, p_node)};
+                    values = make_shared<ngraph::opset1::ReduceSum>(
+                        values,
+                        ngraph::opset1::Constant::create(
+                            element::i64, Shape{reduction_axes.size()}, reduction_axes.to_vector()),
+                        false);
+
+                    shared_ptr<Node> bias_node{ngraph::opset1::Constant::create(
```
Comment (arogowie-intel) [inline src/ngraph/builder/norm.cpp]: Similarly as above: we can create just a scalar.

File: src/ngraph/builder/norm.cpp [hunk 3]
```diff
@@ -36,125 +37,147 @@ namespace ngraph
...
+                        values->get_element_type(),
+                        values->get_shape(),
+                        vector<float>(shape_size(values->get_shape()), bias))};
+
+                    values = make_shared<ngraph::opset1::Add>(values, bias_node);
+
+                    // Get outer part of equation: raise values to 1/p_norm exponent.
+                    shared_ptr<Node> inv_p_node = ngraph::opset1::Constant::create(
```
Comment (arogowie-intel) [inline src/ngraph/builder/norm.cpp]: As above.

File: src/ngraph/builder/norm.cpp [hunk 4]
```diff
@@ -36,125 +37,147 @@ namespace ngraph
...
+                value.get_shape(),
+                vector<float>(shape_size(value.get_shape()), 0.f))};
 
             // Convert bool values to input node data type.
-            shared_ptr<Node> non_zero_values = make_shared<op::Convert>(
-                make_shared<op::NotEqual>(value, zero_node), value.get_element_type());
-
-            return make_shared<op::Sum>(non_zero_values, reduction_axes)
+            const shared_ptr<Node> non_zero_values = make_shared<ngraph::opset1::Convert>(
+                make_shared<ngraph::opset1::NotEqual>(value, zero_node), value.get_element_type());
```
Comment (arogowie-intel) [inline src/ngraph/builder/norm.cpp]: We can relay on NumPy autobroadcasting mechanism for `zero_node` arg.

File: src/ngraph/builder/norm.cpp [hunk 5]
```diff
@@ -36,125 +37,147 @@ namespace ngraph
...
-                op::Constant::create(values->get_element_type(),
-                                     values->get_shape(),
-                                     vector<float>(shape_size(values->get_shape()), bias))};
+            const shared_ptr<Node> bias_node{ngraph::opset1::Constant::create(
+                values->get_element_type(),
+                values->get_shape(),
+                vector<float>(shape_size(values->get_shape()), bias))};
 
-            return (values + bias_node)->add_provenance_group_members_above({value});
+            return make_shared<ngraph::opset1::Add>(values, bias_node)
```
Comment (arogowie-intel) [inline src/ngraph/builder/norm.cpp]: We can relay on NumPy autobroadcasting mechanism for `bias_node` arg.

File: src/ngraph/builder/norm.cpp [hunk 6]
```diff
@@ -36,125 +37,147 @@ namespace ngraph
...
+                    value, value, ngraph::op::AutoBroadcastType::NONE),
+                ngraph::opset1::Constant::create(
                     element::i64, Shape{reduction_axes.size()}, reduction_axes.to_vector()),
                 keep_dims)};
 
-            shared_ptr<Node> bias_node{
-                op::Constant::create(values->get_element_type(),
-                                     values->get_shape(),
-                                     vector<float>(shape_size(values->get_shape()), bias))};
+            shared_ptr<Node> bias_node{ngraph::opset1::Constant::create(
```
Comment (arogowie-intel) [inline src/ngraph/builder/norm.cpp]: We can relay on NumPy autobroadcasting mechanism.

File: src/ngraph/builder/norm.cpp [hunk 7]
```diff
@@ -36,125 +37,147 @@ namespace ngraph
...
+                values->get_shape(),
+                vector<float>(shape_size(values->get_shape()), bias))};
             shared_ptr<Node> result;
             switch (bias_mode)
             {
             case BiasMode::MAX:
             {
-                result = make_shared<op::Sqrt>(make_shared<op::Maximum>(values, bias_node));
+                result =
+                    make_shared<ngraph::opset1::Sqrt>(make_shared<op::Maximum>(values, bias_node));
```
Comment (tsocha) [inline src/ngraph/builder/norm.cpp]: Would you kindly use `opset1::Maximum` here?

File: src/ngraph/builder/norm.cpp (hunk 1 above)
Comment (mbencer) [reply src/ngraph/builder/norm.cpp]: done

File: src/ngraph/builder/norm.cpp (hunk 2 above)
Comment (mbencer) [reply src/ngraph/builder/norm.cpp]: done

File: src/ngraph/builder/norm.cpp (hunk 3 above)
Comment (mbencer) [reply src/ngraph/builder/norm.cpp]: done

File: src/ngraph/builder/norm.cpp (hunk 4 above)
Comment (mbencer) [reply src/ngraph/builder/norm.cpp]: done

File: src/ngraph/builder/norm.cpp (hunk 5 above)
Comment (mbencer) [reply src/ngraph/builder/norm.cpp]: done

File: src/ngraph/builder/norm.cpp (hunk 6 above)
Comment (mbencer) [reply src/ngraph/builder/norm.cpp]: done

File: src/ngraph/builder/norm.cpp (hunk 7 above)
Comment (mbencer) [reply src/ngraph/builder/norm.cpp]: thanks, done

**Pull request merged**